src/board.js

```diff
@@ -0,0 +1,20 @@
+class Board {
+  constructor() {
+    this.gameBoard = ['', '', '', '', '', '', '', '', ''];
+    this.boardCount = 0;
+  }
+
+  // updateBoard(mark, position){
+  //   this.gameBoard[position] = mark;
+  //   this.boardCount++;
+  // }
+
+  // boardFull(){
+  //   if (this.boardCount == this.gameBoard.length) {
+  //     return true;
+  //   } else {
+  //     return false;
+  //   }
+  // }
+}
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**_I trust you guys can do this before merging 👆_**